packages/desktop/components/ColorPicker.svelte

```diff
@@ -0,0 +1,154 @@
+<script lang="ts">
+    import { Text, Icon, Tooltip, TextType, Position } from '@ui'
```

Comment:
Cant we try a way to have this component in shared and be used by desktop & mobile? I feel like this component is basically the same in both platforms, mobile is the same component except for:
- mobile doesnt have tooltips
- mobile doesnt have custom colors, just the pre-defined ones
If possible, I think we should try to avoid code duplication 🙏🏼